spotbugs/etc/messages.xml

```diff
@@ -1729,6 +1729,13 @@ factory pattern to create these objects.</p>
       ]]>
     </Details>
   </Detector>
+  <Detector class="edu.umd.cs.findbugs.detect.CollectionIteratorCheck">
+    <Details>
+      <![CDATA[
+        <p>The method of collection is wrong.</p>
```

Comment:
This description doesn't make proper sense. Should it be something like "Wrong method called on a Collection"?